exampleCourse/questions/examplesMatrixOutput/server.py

```diff
@@ -0,0 +1,23 @@
+import random, math
+
+def generate(data):
+
+    # Simulate values
+    a = random.randint(2, 10)
+    b = random.randint(2, 10)
+    c = random.randint(20,100)/100
```

Comment:
maybe we should add an example that's just a random real number, so it has lots of digits displayed by default?

Reply:
Added this, but the default display is still not acting as expected.